training.php

```diff
@@ -59,6 +59,11 @@ function training() {
 				$train_length = $stat_extended_train_length;
 				$train_gain = $stat_extended_train_gain - 20;
 			}
+			// TEAM BOOST TRAINING GAINS
+			if ($player->team['boost'] == 'Training') {
+                $percent_increase = ceil($train_gain * $player->team['boost_amount'] / 100);
+                $train_gain += $percent_increase;
```

Comment:
Hmm this one's a bit tricky since we have whole-number stats. I think I'm in favor of sticking with the time-based approach here since it scales to any level of % boost without needing to be concerned with rounding

Reply:
I was partially conflicted on this. I just think it's a bit simpler to not mess with the training timers since it's more inconvenient to time your training and such when it's not an irregular amount. Like dealing with 5 hours 24 minutes instead of 6 hours, or 4mins 15secs instead of 5 minutes. In my opinion, 5 minutes is already pretty short lol.

Reply:
Yeah, the problem is currently with a 2.5% training boost and an 8 point gain, you get 8.2 which rounds to 8.0

Reply:
Alternatively can do % chance to get 1 extra stat point (but this needs to be on training completion, so that people can rapidly restart trainings)